src/tools/remote-test-client/tests/lib.rs

```diff
@@ -8,3 +8,18 @@ fn test_help() {
     let stdout = String::from_utf8(output.stdout.clone()).unwrap();
     assert!(stdout.trim().starts_with("Usage:"));
 }
+
+#[test]
+fn test_timeout() {
+    let mut cmd = assert_cmd::cargo::cargo_bin_cmd!();
+    cmd.env("TEST_DEVICE_CONNECT_TIMEOUT_SECONDS", "1");
+    cmd.env("TEST_DEVICE_ADDR", "0.0.0.0:6969");
```

Comment:
let's use a valid destination address to not distract from that it is the timeout we want to test (afaik 0.0.0.0 is never a valid connection ip)


```suggestion
    cmd.env("TEST_DEVICE_ADDR", "127.69.69.69:6969");
```